boa/core/recipe_output.py

```diff
@@ -602,7 +602,7 @@ def _solve_env(self, env, all_outputs):
                 subdir = self.config.build_subdir
 
             solver, pkg_cache = get_solver(
-                subdir, output_folder=self.config.output_folder
+                subdir, self.config, output_folder=self.config.output_folder
```

Comment:
can you pass in the prefix directly here? e.g. either `host` or `build` prefix.

Reply:
Done